server/src/modules/stats/board/board.stats.service.ts

```diff
@@ -0,0 +1,157 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model, Types } from 'mongoose';
+import { BoardStats } from 'src/schemas/stats/board.stats.schema';
+
+@Injectable()
+export class BoardStatsService {
+  constructor(
+    @InjectModel(BoardStats.name)
+    private readonly boardStatsModel: Model<BoardStats>,
+  ) {}
+
+  async initStats(boardId: string): Promise<void> {
+    await this.boardStatsModel.create({ boardId: boardId });
```

Comment:
`await this.boardStatsModel.create({ boardId: boardId });` = `await this.boardStatsModel.create({ boardId });`

its called shorthand property names

https://stackoverflow.com/questions/38948306/what-is-javascript-shorthand-property

![image](https://github.com/user-attachments/assets/c2a420fa-461d-4d3d-ac95-d4f466761241)